src/modals/Menu.tsx

```diff
@@ -77,10 +80,49 @@ const LanguageSwitcher = () => {
   );
 };
 
+const ThemeSwitcher = () => {
+  const { t } = useTranslation();
+  const { theme, updateTheme } = useContext(ThemeContext);
+
+  const themesList = [];
+  for (const item of ["basic", "dark"]) {
```

Comment:
Call this theme dark(beta), because we have some problems with it for now

Reply:
Fixed